tests/visco_plastic_diffusion.prm

```diff
@@ -0,0 +1,105 @@
+# Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 0
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Max nonlinear iterations               = 1
+set Output directory                       = visco_plastic_dislocation
```

Comment:
Setting the output directory is not necessary for the tests and is ignored. Maybe cleaner to remove it for all the tests?

Reply:
@anne-glerum - Agree. Should I just go ahead and remove it for all of the visco_plastic tests or just these three?